### PR TITLE
Replace (some) of First & Last Name variable example references

### PR DESCRIPTION
**Pull request by hergaiety**
## Issue
https://github.com/ember-learn/guides-source/issues/1480

TL;DR Replacing `firstName` and `lastName` example code with `givenName` or `familyName`, or `title` and `name`, or just simply `name`, or something else entirely for i18n/W3C/respect concerns.

## Pages touched in this batch of changes

* [X] Configuration - Optional Features - default-async-observers
* [X] Ember Data - Creating, Updating and Deleting - Updating Records
* [X] Ember Data - Customizing Serializers - Attribute Names
* [X] Testing - Testing Basics - Stubbing an Object
* [X] Accessibility - Component Considerations - Accessible name - Adding a label to an input element
* [X] Components - Helper Functions - Built-in Helpers - The let helper
* [X] Components - Built-in Components - <Input/> - Setting attributes on Input
* [X] Components - Built-in Components - <Textarea />
* [X] Models - Introduction - Models

## Notes

I don't feel this needs to go back prior to `/release` but I'm open to a suggested amount of backporting.

Open to suggested alternatives for these changes as well.

### CODE REVIEW

File: guides/release/configuring-ember/optional-features.md [hunk 1]
```diff
@@ -196,10 +196,10 @@ can be set to run asynchronously using the `sync: false` option.
 import { observer } from '@ember/object';
 
 Person.extend({
```
Comment (boydkr) [inline guides/release/configuring-ember/optional-features.md:198]: maybe `Image`, or `Screen`, to go better with the aspect ratio theme

File: guides/release/components/built-in-components.md [hunk 2]
```diff
@@ -142,27 +142,27 @@ you will always need to either define the event name in camelCase format (e.g. `
...
 ```
 
 Internally, `<Input @type="checkbox" />` creates an instance of Checkbox. Do *not* use `Checkbox` directly.
 
 ## `<Textarea />`
 
 ```handlebars
-<label for="firstname">First Name</label>
-<Textarea @value={{this.name}} cols="80" rows="6" id="firstname" />
+<label for="textarea-post-name">Post:</label>
```
Comment (ijlee2) [inline guides/release/components/built-in-components.md:161]:
```suggestion
<label for="textarea-post">Post:</label>
```

File: guides/release/components/built-in-components.md [hunk 3]
```diff
@@ -193,8 +193,8 @@ and [`{{mut}}`](https://api.emberjs.com/ember/release/classes/Ember.Templates.he
 in conjunction like shown in the following example:
 
 ```handlebars
-<label for="firstname">First Name</label>
-<Input @value={{mut (get this.person this.field)}} id="firstname" />
+<label for="name">Name:</label>
+<Input @value={{mut (get this.person this.field)}} id="name" name="name" />
```
Comment (ijlee2) [inline guides/release/components/built-in-components.md:197]: Can we continue to use `input-name` for ID here?

```suggestion
<label for="input-name">Name:</label>
<Input @value={{mut (get this.person this.field)}} id="input-name" name="name" />
```

File: guides/release/configuring-ember/optional-features.md (hunk 1 above)
Comment (ijlee2) [reply guides/release/configuring-ember/optional-features.md:198]: Yeah, I think the name can be changed. I'm not sure which of the two better illustrates a synchronous observer.

File: guides/release/configuring-ember/optional-features.md [hunk 4]
```diff
@@ -196,10 +196,10 @@ can be set to run asynchronously using the `sync: false` option.
...
-  partOfNameChanged: observer({
-    dependentKeys: ['firstName', 'lastName'],
+  aspectRatioChanged: observer({
```
Comment (ijlee2) [inline guides/release/configuring-ember/optional-features.md:199]: I think it's a bit odd that the observer name is `aspectRatioChanged`, yet the dependent keys are `width` and `height` (and not `aspectRatio`).

What do you think of naming the observer `onImageSizeChange` or `onScreenSizeChange`?

File: guides/release/models/creating-updating-and-deleting-records.md [hunk 5]
```diff
@@ -19,9 +19,9 @@ Making changes to Ember Data records is as simple as setting the attribute you
 want to change:
 
 ```javascript
-this.store.findRecord('person', 1).then(function(tyrion) {
+this.store.findRecord('post', 1).then(function(post) {
   // ...after the record has loaded
-  tyrion.firstName = 'Yollo';
+  post.title = 'UPDATED: Rails is Omakase';
```
Comment (ijlee2) [inline guides/release/models/creating-updating-and-deleting-records.md:24]: In the subsection `Persisting Records`, the code shows,

```javascript
store.findRecord('post', 1).then(function(post) {
  post.title; // => "Rails is Omakase"

  post.title = 'A new post';

  post.save(); // => PATCH to '/posts/1'
});
```

For continuity, can we either use `A new post` in your line 24, or replace `A new post` with `UPDATED: Rails is Omakase`?

File: guides/release/testing/unit-testing-basics.md [hunk 6]
```diff
@@ -233,8 +233,8 @@ module('Unit | Service | employees', function(hooks) {
     const someThing = this.owner.lookup('service:some-thing');
 
     class MockPerson {
-      firstName = 'John';
-      lastName = 'Smith';
+      title = 'Dr.';
+      name = 'Zoey';
```
Comment (ijlee2) [inline guides/release/testing/unit-testing-basics.md:237]: Line 243 needs to be updated to say `Dr. Zoey is now an employee`.

Comment (ijlee2) [reply guides/release/testing/unit-testing-basics.md:237]: As an aside (this should be addressed in a separate PR), I don't think the assertion in line 244 would work at all. `someThing.employees[0]` and `person` are both class instances. We should probably remove this line.

**Review by ijlee2 — CHANGES_REQUESTED** [review]
@hergaiety Thank you for working on addressing the name issues!

I think a few areas need minor updates. Can you also rebase the branch so that Percy may show 9 snapshots (corresponding to 9 file changes in `release`) to be different?

**Comment by hergaiety** [comment]
All great comments, when I find time I'll address suggested changes. Cheers y'all!

File: guides/release/configuring-ember/optional-features.md (hunk 1 above)
Comment (hergaiety) [reply guides/release/configuring-ember/optional-features.md]: Addressed in 39a216a8

File: guides/release/configuring-ember/optional-features.md (hunk 4 above)
Comment (hergaiety) [reply guides/release/configuring-ember/optional-features.md]: Addressed in 39a216a8

File: guides/release/models/creating-updating-and-deleting-records.md (hunk 5 above)
Comment (hergaiety) [reply guides/release/models/creating-updating-and-deleting-records.md]: Addressed in 39a216a8

File: guides/release/testing/unit-testing-basics.md (hunk 6 above)
Comment (hergaiety) [reply guides/release/testing/unit-testing-basics.md:237]: Addressed in 39a216a8

Will open a separate PR for line 244's assertion.

**Comment by hergaiety** [comment]
@ijlee2 I believe I've rebased this correctly, let me know if there are still issues (feel free to Discord DM me as well).

File: guides/release/testing/unit-testing-basics.md (hunk 6 above)
Comment (hergaiety) [reply guides/release/testing/unit-testing-basics.md:237]: PR opened:  #1535

**Comment by ijlee2** [comment]
Thanks! I can review the changes again now.
